query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -12,7 +12,7 @@ import filodb.core.metadata.Column.ColumnType
 import filodb.core.metadata.Dataset
 import filodb.core.query._
 import filodb.memory.data.ChunkMap
-import filodb.memory.format.{RowReader, UnsafeUtils, ZeroCopyUTF8String}
+import filodb.memory.format.{vectors => bv, RowReader, UnsafeUtils, ZeroCopyUTF8String}
```

Comment:
Style request: Aliasing makes the code harder to read, but it looks like 'bv' is only used later. Can you put the import closer to its first usage? It makes things easier to read, in my opinion.

Reply:
Sure.